src/components/form/range/types.ts

```diff
@@ -201,21 +202,7 @@ export interface EuiDualRangeProps
   /**
    * Intended to be used with aria attributes. Some attributes may be overwritten.
    */
-  minInputProps?: Omit<
-    InputHTMLAttributes<HTMLInputElement>,
-    | 'max'
-    | 'min'
-    | 'value'
-    | 'step'
-    | 'disabled'
-    | 'readonly'
-    | 'name'
-    | 'onChange'
-    | 'onFocus'
-    | 'onBlur'
-    | 'onKeyDown'
-    | 'onMouseDown'
-  >;
+  minInputProps?: Partial<Omit<EuiRangeInputProps, 'max' | 'min' | 'step'>>;
```

Comment:
Nice abstraction! It looks like we may want to omit a few display related props as well that come after `...rest`:
```suggestion
  minInputProps?: Partial<Omit<EuiRangeInputProps, 'max' | 'min' | 'step' | 'compressed' | 'autoSize' | 'fullWidth' | 'controlOnly'>>;
```